### DISCUSSION TITLE
How to create vectors in ordered set notation in KaTeX?

### DISCUSSION THREAD

**Discussion by Ahsoka** [KaTeX/KaTeX#3644] [Q&A] [answered]
Hello,

I was wondering how to create vectors in ordered set notation.

See the following image for reference:
![Sample Ordered Set Notation](https://wikimedia.org/api/rest_v1/media/math/render/svg/081296ac427aa7441e0f79b9eaea7d88f9c41c73).

(if you are on dark mode you may not be able to view the image, please see the image against a white background to see the rendered LaTeX)

Image from: https://en.wikipedia.org/wiki/Vector_notation#Ordered_set_notation

**Comment by edemaine** [comment, marked as answer]
If you view the source on that Wikipedia page, you see the relevant LaTeX:
```latex
\mathbf{v} = \langle v_1, v_2, \dots, v_{n - 1}, v_n \rangle
```
That's how I'd normally write angled brackets like that.  You could also use `\left\langle ... \right\rangle` to get larger forms.

**Reply by Ahsoka** [reply]
Hey there! Thank you for the response.  How do you view the source LaTeX on Wikipedia?  I will try to do that first whenever I need to figure out how something was made on there.  Also do you know what typesetting package Wikipedia uses?

Thank you for all your help!

**Comment by mbourne** [comment]
This page answers all: https://en.wikipedia.org/wiki/Help:Displaying_a_formula
